libraries/rush-lib/package.json

```diff
@@ -54,7 +55,9 @@
     "strict-uri-encode": "~2.0.0",
     "tapable": "2.2.1",
     "tar": "~6.1.11",
-    "true-case-path": "~2.2.1"
+    "true-case-path": "~2.2.1",
+    "npm-check": "~5.9.2",
+    "text-table": "~0.2.0"
```

Comment:
We already have "print in table" code [here](https://github.com/microsoft/rushstack/blob/main/libraries/rush-lib/src/cli/actions/ListAction.ts#L181-L258). Can you refactor that and reuse it instead of pulling in a new dependency.

Reply:
For MVP I'd like to keep text-table, however I would definitely refactor our print in table code, or extend it to be colorized/useful in this scenario.

Reply:
There's also `cli-table`. Can that be reused?

Reply:
Alright I've switched to cli-table. It's pretty identical results besides some tweaking with column widths.